service/resource/flanneldv1/create.go

```diff
@@ -2,12 +2,61 @@ package flanneldv1
 
 import (
 	"context"
+
+	"github.com/giantswarm/microerror"
+	"k8s.io/api/extensions/v1beta1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+
+	"github.com/giantswarm/flannel-operator/service/keyv2"
 )
 
 func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange interface{}) error {
+	customObject, err := keyv2.ToCustomObject(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+	daemonSetToCreate, err := toDaemonSet(createChange)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	if daemonSetToCreate != nil {
+		r.logger.Log("cluster", keyv2.ClusterID(customObject), "debug", "creating the daemon set in the Kubernetes API")
+
+		namespace := keyv2.NetworkNamespace(customObject)
+		_, err = r.k8sClient.Extensions().DaemonSets(namespace).Create(daemonSetToCreate)
+		if apierrors.IsAlreadyExists(err) {
+			// fall through
```

Comment:
Would it be an error if the daemonset existed already? I don't see how that could happen.

Reply:
Uncertain. I think this is just a safety net? We have this like in each resource.

Reply:
Good enough for me. Thought I'd bring it up.

Reply:
Appreciated!